src/coreclr/pal/src/libunwind/src/CMakeLists.txt

```diff
@@ -342,7 +342,7 @@ else(CLR_CMAKE_HOST_UNIX)
         list(APPEND libunwind_setjmp_la_SOURCES     x86_64/longjmp.S x86_64/siglongjmp.SA)
     endif()
 
-    set_source_files_properties(${CLR_DIR}/pal/src/exception/remote-unwind.cpp PROPERTIES COMPILE_FLAGS /TP INCLUDE_DIRECTORIES ${CLR_DIR}/inc)
```

Comment:
Would it be possible to modify the file to not require c++latest? If it is just the struct initialization, it seems it would be easy. I am a bit nervous about mixing c++ standards.

Reply:
Since this was blocking dev inner loop, I thought this could be merged as is to unblock people, then fix this next week when I have more time and direct access to my Windows machine.

Reply:
If someone else can fix the compile issues, that works too.

Reply:
I guess looking at it, it looks trivial to fix...  I'll amend this

Reply:
Done